Client/Coordinators/Library/LibraryCoordinator.swift

```diff
@@ -18,22 +18,22 @@ protocol LibraryNavigationHandler: AnyObject {
 class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate, LibraryNavigationHandler {
     private let profile: Profile
     private let tabManager: TabManager
-    private let libraryViewController: LibraryViewController
+    private var libraryViewController: LibraryViewController!
     weak var parentCoordinator: LibraryCoordinatorDelegate?
 
     init(
         router: Router,
         profile: Profile = AppContainer.shared.resolve(),
         tabManager: TabManager = AppContainer.shared.resolve()
     ) {
-        self.libraryViewController = LibraryViewController(profile: profile, tabManager: tabManager)
         self.profile = profile
         self.tabManager = tabManager
         super.init(router: router)
-        self.router.setRootViewController(libraryViewController)
     }
 
     func start(with homepanelSection: Route.HomepanelSection) {
+        libraryViewController = LibraryViewController(profile: profile, tabManager: tabManager)
+        router.setRootViewController(libraryViewController)
```

Comment:
So I'm afraid by recreating the library VC we're going to lose some of the states saved in for example the bookmarks panel. Can you check this still works (the section we're in should be kept). Hence why this was saved and LibraryCoordinator isn't niled after the user leaves the library panels.